libs/platform/src/lib/table/components/table-header-row/table-header-row.component.html

```diff
@@ -80,7 +80,7 @@
                     : _tableColumnResizeService.getColumnWidthStyle(column.name)
                     : _tableColumnResizeService.getNextColumnsWidth(column.name)
         "
-        [fdPopoverTrigger]="tablePopover.popover"
+        [fdPopoverTrigger]="_isColumnHasHeaderMenu(column) ? tablePopover.popover : null"
```

Comment:
avoid using functions in template. Better to introduce pipe (possibly `pure` will work too)